docs/carbon-as-spark-datasource-guide.md

```diff
@@ -15,19 +15,20 @@
     limitations under the License.
 -->
 
-# Carbon as Spark's datasource guide
+# CarbonData as Spark's Datasource
 
-Carbon fileformat can be integrated to Spark using datasource to read and write data without using CarbonSession.
+The CarbonData fileformat is now integrated as Spark datasource for read and write operation without using CarbonSession. This is useful for users who wants to use carbondata as spark's data source. 
 
+**Note:** You can only apply the features of Spark datasource similar to Parquet. The carbon session features are not supported.
```

Comment:
functions/features supported by spark data source APIs.Functionalities supported would be similar to Parquet